front-end/src/app/layout/sidebar/menu-report/menu-report.component.ts

```diff
@@ -161,23 +146,13 @@ export class MenuReportComponent implements OnInit, OnDestroy {
         ];
       }
 
-      // Slice indexes are determined by the number of entries in each urlMatch group
-      this.items[0].expanded = this.isActive(this.urlMatch.slice(0, 4), event.url);
-      this.items[1].expanded = this.isActive(this.urlMatch.slice(4, 7), event.url);
-      this.items[2].expanded = this.isActive(this.urlMatch.slice(7, 10), event.url);
-    }
-  }
+      this.items[0].expanded = this.sidebarState == ReportSidebarState.TRANSACTIONS;
+      this.items[1].expanded = this.sidebarState == ReportSidebarState.REVIEW;
+      this.items[2].expanded = this.sidebarState == ReportSidebarState.SUBMISSION;
 
-  /**
-   * Determine if the given url matches one of the regular expressions that define which
-   * group of menu items is selected.
-   *
-   * @param urlMatch {RegExp{}} - List of url regular expressions to compare to current router url
-   * @param url {string} - Current url in browser address bar
-   * @returns {boolean} - True is url matches one of the matchUrl regular expressions
-   */
-  isActive(urlMatch: RegExp[], url: string): boolean {
-    return urlMatch.reduce((prev: boolean, regex: RegExp) => prev || regex.test(url), false);
+      // This fixes a bug where the sidebarState is undefined when first navigating to a report
+      if (this.sidebarState == undefined) this.items[0].expanded = true;
```

Comment:
could you just initialize `items[0].expanded = true` at the top of the class?

Reply:
the items are defined immediately prior to this, so they can't, unfortunately.  It also can't be initialized to true without needing to be set to false again if another section is supposed to be expanded.  Now that the component will only be loaded when there's a relevant report, there's room to rewrite the whole component, but I think that it would be better to wait on that until there's a second sidebar to implement that could share code with it.